python/tester/upload.py

```diff
@@ -56,12 +56,14 @@
 class Instrument:
     n_snaps: int
     n_detectors: int
+    sal_index: int = 0
 
 
 INSTRUMENTS = {
-    "LSSTCam": Instrument(2, 189 + 8 + 8),
-    "LSSTComCam": Instrument(2, 9),
-    "LATISS": Instrument(1, 1),
+    "LSSTCam": Instrument(2, 189 + 8 + 8, 1),
+    "LSSTComCam": Instrument(2, 9, 3),
```

Comment:
Isn't it more likely that the real ComCam will be on queue 1, given that that's the main telescope?

Reply:
Yes. Thank you for spotting it!